Clarify GitHub and GitLab setup in Slack agent docs (#5661)
mintlify/docs: agent/slack.mdx

> Once your Slack workspace is connected, you can ask the agent in Slack to set up any third-party integration that it should use as context. The agent uses [Composio](https://docs.composio.dev/toolkits) to support a broad catalog of third-party services, including issue trackers, knowledge bases, CRMs, and developer tools.

Reviewer: ```suggestion
Once you connect your Slack workspace, you can ask the agent in Slack to set up any third-party integration as context. The agent uses [Composio](https://docs.composio.dev/toolkits) to support a broad catalog of third-party services, including issue trackers, knowledge bases, CRMs, and developer tools.
```

<Note>

>   GitHub and GitLab are not configured through Composio. Repository access is handled by the [Mintlify GitHub App](/deploy/github) or [GitLab integration](/deploy/gitlab) installed from your dashboard. Once installed, the agent can read source code from any repository the app has access to.

Reviewer: ```suggestion
  You must configure GitHub and GitLab through Mintlify, not Composio. Use the [GitHub app](/deploy/github) or [GitLab integration](/deploy/gitlab) to control which repositories the agent can access.
```
